packages/mds-schema-validators/validators.ts

```diff
@@ -209,6 +210,42 @@ const auditIssueCodeSchema = stringSchema.max(31)
 
 const auditNoteSchema = stringSchema.max(255)
 
+const vehicleTypesCountMapSchema = Joi.object().keys({
+  scooter: Joi.number(),
+  bike: Joi.number(),
+  carshare: Joi.number()
```

Comment:
Should this include all vehicle types or no?

Reply:
It should indeed -- with the switch of `carshare -> car` and addition of `moped` this needs to be updated